routing.rst

```diff
@@ -995,6 +995,136 @@ convert them automatically to their scalar values.
         }
     }
 
+Mapping Parameters
+~~~~~~~~~~~~~~~~~~
+
+By default, the name of the variable part (``{slug}`` for example) is the
```

Comment:
I think we should not use the terminology _"variable part"_. Instead, let's use "parameter" or "route parameter", because we're in the section that explains route parameters and the reader will find that consistent.